bindata/v3.11.0/kube-apiserver/pod.yaml

```diff
@@ -13,10 +13,21 @@ spec:
     - name: fix-audit-permissions
       image: ${IMAGE}
       imagePullPolicy: IfNotPresent
-      command: ['sh', '-c', 'chmod 0700 /var/log/kube-apiserver']
+      command: ['/bin/sh', '-c', 'chmod 0700 /var/log/kube-apiserver']
       volumeMounts:
         - mountPath: /var/log/kube-apiserver
           name: audit-dir
+    - name: wait-for-port
+      image: ${IMAGE}
+      imagePullPolicy: IfNotPresent
+      command: ['/usr/bin/timeout', '90', "/bin/bash", "-c"]
```

Comment:
does this correlate with the time revision controller is willing to wait for apiserver to be up?

Reply:
90 >> 60 seconds for graceful termination. But I wonder whether the 34 sec extra wait time must be added.

Reply:
updated to 100: 60 for graceful termination, 35 for https://github.com/openshift/cluster-kube-apiserver-operator/pull/318